flowmachine/flowmachine/utils/utils.py

```diff
@@ -240,6 +242,54 @@ def proj4string(conn, crs=None):
     return proj4_string.strip()
 
 
+def verify_columns_exist_in_all_tables(conn, tables, columns):
+    """
+    Parse a list of tables ensuring that certain columns are present.
+
+    Parameters
+    ----------
+    conn : Connection
+        FlowMachine db connection to use to get the subscriber_tables.
+    tables : str or list of strings, default 'all'
+        Can be a sting of a single table (with the schema) or a list of these.
+        The keyword all is to select all subscriber tables
+    columns : str or list
+        A string or list of strings with the column names that must be present.
+
+    Returns
+    -------
+    list
+        Tables that contains the required columns.
+    """
+
+    if isinstance(tables, str) and tables.lower() == "all":
+        tables = [f"events.{t}" for t in conn.subscriber_tables]
+    elif type(tables) is str:
+        tables = [tables]
+    else:
+        tables = tables
+
+    if isinstance(columns, str):
+        columns = [columns]
+
+    parsed_tables = []
```

Comment:
If we remove the return value then the logic in lines 275-290 can be significantly simplified because you don't need to keep track of parsed tables:
```
for t in tables:
    for c in columns:
        if c not in flowmachine.core.Table(t).column_names:
            raise MissingColumnsError(tables_lacking_columns, columns)
```